app/javascript/pages/base.scss

```diff
@@ -244,4 +244,9 @@ $colors: (
   }
 }
 
-@import "tailwindcss/utilities";
+// Override bloom styles so that navbar doesn't overflow on medium screens.
```

Comment:
@software-project can you re-review this? this feels really hacky but is the only way I could figure out how to avoid this overflow issue. Do you have any other ideas?
![image (45)](https://user-images.githubusercontent.com/11825994/129102011-e6cb090d-4a31-4b50-bf20-6c9b9114b83c.png)

Reply:
Well, I tried to play with shrink and frow, but all in all it got me to the same result. I would assume that those properties were designed to serve exactly that purpose.